hs_core/models.py

```diff
@@ -250,27 +309,45 @@ def page_permissions_page_processor(request, page):
 
     show_manage_access = False
     if not cm.raccess.published and \
-            (is_owner_user or (cm.raccess.shareable and (is_view_user or is_edit_user))):
+            (self_access_level == 'owner' or (
+                cm.raccess.shareable and (self_access_level == 'view' or self_access_level == 'edit'))):
         show_manage_access = True
 
     return {
         'resource_type': cm._meta.verbose_name,
         'bag': cm.bags.first(),
-        "edit_users": editors,
-        "view_users": viewers,
+        "users_json": users_json,
         "owners": owners,
-        "edit_groups": edit_groups,
-        "view_groups": view_groups,
-        "is_owner_user": is_owner_user,
-        "is_edit_user": is_edit_user,
-        "is_view_user": is_view_user,
+        "self_access_level": self_access_level,
         "can_change_resource_flags": can_change_resource_flags,
         "is_replaced_by": is_replaced_by,
         "is_version_of": is_version_of,
         "show_manage_access": show_manage_access
     }
 
 
+# TODO: import this from hydroshare_tags
+def best_name(content):
```

Comment:
@cuahsimarko - I was unnable to import this method from hydroshare_tags. Perhaps you could give it a try.

Reply:
When the "Dashboard" was pushed out, a new hs_docker_image was built. Please work with Scott or Phuong to engage with the 1.9.8 base image.

Reply:
Not at all, I don't think this issue had to do with "timeago" or outdated image.

Reply:
After discussing with you, I'm still not sure why the import statement you tried isn't working. @sblack-usu is anything jumping out at you why from hs_core... import best_name isn't working for Mauriel here?

Reply:
I was able to import without any trouble, what does your import look like?  It should be something like this
`from .templatetags.hydroshare_tags import best_name`

Reply:
Here is the error we are getting:
![image](https://user-images.githubusercontent.com/2448568/58340894-2c543680-7e0a-11e9-9e00-3df8d19a4bb9.png)

Were you able to import it in this file and running this branch?

Reply:
So the problem is a circular import.  If you look at the last import in the stacktrace, you can see it's trying to make imports from hs_core.models, which is back to where you're stating the import for best_name.

You can get around this by limiting the scope of where you make this import by importing just before you need it, so line 241 seems like a good place to put this, `from templatetags.hydroshare_tags import best_name`